ethereum/block/testdata/generate-bindings.sh

```diff
@@ -1,9 +1,9 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
Macos 12 has a legacy version of bash installed (3.2 from 2007) and they plan on removing bash entirely (default shell is now zsh). All the scripts were adjusted so that the first `bash` executable in the user's path is used. You can easily install bash 5.2 using brew on macos.